### ISSUE TITLE
Hobbykommittéer med egen ekonomi

### ISSUE THREAD

**Issue by Kirens** [dtekcth/styrdokument#18]
Vill minnas det som att idén med omstruktureringen av kommittér var att alla med egen ekonomi skulle vara sektionskommittéer. Ekonomiskar reglementets §4.2 refererar till hobbykommittéer med egen ekonomi. Är detta bara en gammal relik kvar i dokumentet tro?

https://github.com/dtekcth/styrdokument/blob/868f183bcfd4f0c1b88e78636430bee059fc9f51/ekonomiskt_reglemente/ekonomiskt_reglemente.tex#L155

**Comment by emilhem** [comment]
Hmm, bra fråga. Minns inte exakt.

**Comment by Rembane** [comment]
Det ska finnas någon form av förarbete eller mer information i frågan i sektionsmötesprotokoll från när @emilhem var sektionskassör. I slutet av året är högre chans än början av året.

**Comment by emilhem** [comment]
Kanske @annabru vet mer?

**Comment by annabru** [comment]
Yes, tanken med omstrukturering var att man skulle ha koll på vilka som behövde redovisa sin  ekonomi /verksamhet. 

Vet att det finns lite planer på att låta sektkass sköta all bokföring (och bli arvoderad), så detta kanske kommer vilja ändras. Men gött tänker jag att ha det uppdelat i reglementet vilka man kräver verksamhetsrapport från 🙂
